File: Valghalla.Application/Communication/CommunicationLogDetails.cs
namespace Valghalla.Application.Communication
{
    public class CommunicationLogDetails
    {
        public Guid Id { get; init; }
        public Guid ParticipantId { get; init; }
        public string ParticipantName { get; init; } = null!;
        public CommunicationLogMessageType MessageType { get; init; } = null!;
        public CommunicationLogSendType SendType { get; init; } = null!;
        public string Message { get; init; } = null!;
        public string ShortMessage { get; init; } = null!;
        public CommunicationLogStatus Status { get; init; } = null!;
        public string? Error { get; init; }
        public DateTime CreatedAt { get; init; }
    }
}

File: Valghalla.Internal.Web/src/features/communication/communication-log/models/communication-log-details.ts
import { CommunicationLogMessageType } from "./communication-log-message-type";
import { CommunicationLogSendType } from "./communication-log-send-type";
import { CommunicationLogStatus } from "./communication-log-status";

export interface CommunicationLogDetails {
  id: string;
  participantId: string;
  participantName: string;
  messageType: CommunicationLogMessageType;
  sendType: CommunicationLogSendType;
  message: string;
  shortMessage: string;
  status: CommunicationLogStatus;
  error?: string;
  createdAt: Date;
}
